src/pages/console/formbuilder/overview.mdx

```diff
@@ -0,0 +1,138 @@
+
+</Callout>
+
+For each data model, Amplify Studio will generate two forms: a **Create** form, and an **Update** form. 
+
+Your forms will contain inputs for each field of your data model. Auto-generated forms are dynamic; as your data model changes, the forms' fields will update accordingly.
+
+## Option 2: Build a form from scratch
+
+Alternatively, Amplify Studio allows you to build forms completely from scratch. To generate a basic, brand-new form:
+
+1. Navigate to **Studio Console > UI Library**
+2. In the left-hand nav bar, next to the **Forms** header, select **New**
+3. Give your form a name
+4. For this example, leave the other options as default, and click **Create form**
+
+![New form page, showing the options available when creating a new form](/images/console/formbuilder/overview2.png)
+
+After clicking **Create form**, you will be directed to the Edit Form page. Select the **plus sign**, and select an input element to add to your form. 
+
+![Image of the Form Creation page with the blue bar and plus sign indicating the ability to add input elements to the form](/images/console/formbuilder/overview3.png)
+
+Alternatively, you can select different options when generating your form:
+- Forms created as **Update** forms will expect input data
+- Forms that start with a **Data model** will have an input element for every field on the data model, similar to an auto-generated form
+- Learn more [JSON object forms below](#option-3-generate-a-form-from-json)
+
+## Option 3: Generate a form from any JSON object
+
+Amplify Studio can generate a form based on the JSON object you'd like it to produce. This is particularly useful if you are interfacing with another API - most API endpoints expect input data in JSON format, so you can quickly generate a form by pasting a sample payload. 
+
+To generate a form from JSON:
+1. Navigate to **Studio Console > UI Library**
+2. In the left-hand nav bar, next to the **Forms** header, select **New**
+3. In the New Form page, select **JSON object**
+4. Paste your JSON into the code editor
+5. Give your form a name, and click **Create form**
+
+Form Builder will infer the input element type based on the value, and can handle arrays and nested JSON values. For example, use this JSON sample:
+
+```json
+{
+  "basics": {
+    "firstName": "Wesley",
+    "emailAddress": "wpeck@amazon.com"
+  },
+  "favoriteThings": {
+    "animals": ["Cats", "Snakes", "Quokka"],
+    "month": "December",
+    "number": 17
+  },
+  "active": true,
+  "enabled": false
+}
+```
+
+This JSON will render the following form:
+
+![Form rendered from JSON showing headings and fields with types inferred from key pair values](/images/console/formbuilder/overview6.png)
+
+From the JSON block, Amplify Studio automatically...
```

Comment:
Either change the items below to `sets` or change this line to `Amplify Studio can automatically`